README.md

```diff
@@ -33,6 +35,91 @@ A collaborative, youth­led project aiming to make use of digital technology to
 1. [Install Git](http://git-scm.com/download/mac)
 2. Clone the repo: `git clone git@github.com:womenhackfornonprofits/ocdaction.git`
 
+
+1. Install [virtualenv](https://virtualenv.pypa.io/en/stable/): 
+
+	```sudo pip install virtualenv```
+
+2. Install [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html):
+
+3. Source the `virtualenvwrapper`:
+	```source /usr/local/bin/virtualenvwrapper.sh```
+	
+	**NOTE**: To help do this automatically on every new shell you open add the line above to your `.bash_profile` or  `.bashrc`
+
+4. Create a new env for the project:
+ 
+	 ```mkvirtualenv ocdaction```
+6. Get the code: 
+
+	```git clone git@github.com:womenhackfornonprofits/ocdaction.git```
+	
+6. Go inside the `ocdaction` directory:
+	
+	```cd ocdaction```
+
+7. Activate the virtual enviroment:
+ 
+	 ```workon ocdaction``` 
+	 
+	 This will now ensure anything you install is within this enviroment.
+
+8. You will need to have [Postgres installed](https://www.postgresql.org/download/) and up and running. You can onstall it via:
+	- Homebrew ```brew install postgresql```
+	- OR Download the [Postgress App](http://postgresapp.com/)
+
+9. Make sure the Postgres Server is up and running:
+	- If using the App simply start the server from there
+	- If using command line: ``brew services start postgresql``
+
+9. Install the requirements:
+
+	 ```pip install -r requirements.txt``` 
+	 
+	 This will get all the dependencies.
+
+9. Create a database locally for the project to run:
+	
+	```createdb ocdaction```
+
+10. Go inside frontend folder: 
+	
+	```cd frontend```
+
+11. Install all the dependencies:
+
+	 ```npm install```
+
+	 This will get all the dependencies
+
+	
+## Running the project locally
+1. Go inside the django app directory: 
+
+	```cd ocdaction```
+2. Run django server:
+	
+	```python manage.py runserver```
```

Comment:
I get `No module named ocdaction.settings`, although I can see the settings file there. Maybe it's my config? I'll check.